doc/enterprise/auth/auth.md

```diff
@@ -0,0 +1,234 @@
+to **Authorize Pachyderm** appears. After you authorize
+Pachyderm, a Pachyderm user token appears:
+
+![alt tag](auth.png)
+
+1. Copy and paste this token back into the Pachyderm login
+screen and press **Enter**. You are now logged in to Pachyderm,
+and you must see your GitHub avatar and an indication of your
+user in the upper left-hand corner of the dashboard:
+
+![alt tag](auth_dash3.png)
+
+
+### Log in to the CLI
+
+To authenticate by using `pachctl`, run the following
+command:
+
+1. Log in with your user name:
+
+   ```bash
+   pachctl auth login <username>
+   ```
+
+   When you run this command, `pachctl` provides
+   you with a GitHub link to authenticate as the provided
+   GitHub user.
+
+   **Example:**
+
+   ```bash
+   $ pachctl auth login dwhitena
+   (1) Please paste this link into a browser:
+
+   https://github.com/login/oauth/authorize?client_id=d3481e92b4f09ea74ff8&redirect_uri=https%3A%2F%2Fpachyderm.io%2Flogin-hook%2Fdisplay-token.html
+
+   (You will be directed to GitHub and asked to authorize Pachyderm's login app on Github. If you accept, you will be given a token to paste here, which will give you an externally verified account in this Pachyderm cluster)
+
+   (2) Please paste the token you receive from GitHub here:
+   ```
+
+   If you have not previously authorized Pachyderm on GitHub, an option
+   to **Authorize Pachyderm** appears. After you authorize Pachyderm,
+   a Pachyderm user token appears:
+
+   ![alt tag](auth.png)
+
+   1. Copy and paste this token back into the terminal and press enter.
+
+   You are now logged in to Pachyderm!
+
+## Manage and update user access
+
+You can manage user access in the UI and CLI.
+For example, when you log in to Pachyderm as the user `dwhitena`, you
+have a repository called `test`.  Because the user `dwhitena` created
+this repository, `dwhitena` has full read and write access to the repo.
+You can confirm this on the dashboard by navigating to or clicking on
+the repo `test`:
+
+![alt tag](auth_dash4.png)
+
+
+Alternatively, you can confirm your access by running the
+`pachctl auth get ...` command.
+
+**Example:**
+
+```
+$ pachctl auth get dwhitena test`
+OWNER
+```
+
+An OWNER of `test` or a cluster admin can then set other user’s
+scope of access to the repo by using
+the `pachctl auth set ...` command or through the dashboard.
+
+For example, to give the GitHub users `JoeyZwicker` and
+`msteffen` `READER`, but not `WRITER` or `OWNER`, access to
+`test` and `jdoliner`, `WRITER`, but not `OWNER`, access,
+click on **Modify access controls** under the repo details
+in the dashboard. This functionality allows you to add
+the users easily one by one:
+
+![alt tag](auth_dash5.png)
+
+## Behavior of pipelines as related to access control
+
+In Pachyderm, you do not explicitly set the scope of access
+for users on pipelines. Instead, pipelines infer access from
+the repositories that are input to the pipeline. Pipelines
```

Comment:
also from the repository that the pipeline outputs to—users need access to both (though for new pipelines, the repository that the pipeline outputs to doesn't exist yet, and you can't have access to a repo that doesn't exist, so we don't require that, but hopefully that's clear enough from context?)

Reply:
You should feel free to edit this however seems best, but here's how I would write this part:

"In Pachyderm, you do not explicitly grant users access to pipelines. Instead, pipelines infer access from their input and output repositories. To update a pipeline, you must have at least `READER`-level access to all pipeline inputs and at least `WRITER`-level access to the pipeline output. This is because pipelines read from their input repos and write to their output repos, and you cannot grant a pipeline more access than you have yourself."

Reply:
I'm a bit confused by "though not sufficient".